modules/ROOT/pages/endpoint-as2-receive.adoc

```diff
@@ -41,11 +42,36 @@ a| The HTTP or HTTPS address where you receive AS2 messages.
 |===
 
 
-== AS2 URL
+== AS2 Receive Endpoint URL
 
-The value in the URL field is automatically generated when you deploy a message flow to CloudHub that uses this endpoint. You can view this URL by selecting the receiving endpoint from your deployed message flow.
+=== Deploy to CloudHub on a Public Port
 
-If you are deploying to an on-premises Mule instance with a firewall and load balancer, work with your organization's hosting or network security team to obtain the externally accessible DNS or the name of the instance host. You must replace the `{runtime_host}` value in the URL with the value that is specific to your organization. For example, if the DNS host name is `b2b.mycompany.com`, the AS2 receiver URL is `+https://b2b.mycompany.com/receive-as2/+`.
+When an AS2 receive endpoint is deployed with the private port configuration enabled, the endpoint application is deployed on the public HTTPS port (8082), with the traffic routed through the xref:runtime-manager::dedicated-load-balancer-tutorial#shared-load-balancers [shared load balancer]. The value in the URL field is automatically generated when you deploy a message flow to CloudHub that uses this endpoint.
+
+You can view this URL by selecting the receiving endpoint from your deployed message flow or by clicking the endpoint name from the endpoints list. You can share this URL with the partners that send AS2 messages to this endpoint.
+
+If your organization uses CloudHub VPC without a dedicated load balancer (DLB), your organization’s platform administrator can configure the necessary xref:runtime-manager::vpc-firewall-rules-concept.adoc[VPC firewall rules] to allow specific IP addresses of your
+partners to access the endpoint.
+
+=== Deploy to CloudHub on a Private Port
+
+When an AS2 receive endpoint is deployed with the private port configuration enabled, the endpoint is deployed to the private HTTPS port (8092) and the internal URL of the endpoint appears on the *Message Flow* or *Endpoint* detail page.
+
+After deploying the first message flow using the AS2 receive endpoint configuration, you can view the runtime application name for the endpoint from the *Endpoint* details page or the *Receive* section of your message flow. To configure xref:runtime-manager/lb-mapping-rules[URL mapping rules] in the xref:runtime-manager::cloudhub-dedicated-load-balancer.adoc[DLB] to forward AS2 messages received from your partners to the application, provide the application name to your organization’s platform administrator.
+
+Your organization’s platform administrator can also xref:runtime-manager::lb-whitelists.adoc[whitelist] your partners’ IP addresses in the DLB settings to allow specific IP addresses to access the endpoint.
```

Comment:
We really can't change the text in [ ] but STET the xref syntax?

xref:runtime-manager::lb-whitelists.adoc[add to a list of allowed addresses]

Reply:
Done - modified slightly to fit the context of the sentence